rosdep/python.yaml

```diff
@@ -731,7 +731,7 @@ python-boto3:
   opensuse: [python-boto3]
   ubuntu:
     '*': [python-boto3]
-    trusty: null
```

Comment:
Same here.

Reply:
Oh yes, did not see that, must have been removed by clean_rosdep_yaml.py. I'll fix it!